modules/router-certificates.adoc

```diff
@@ -0,0 +1,94 @@
+requests a certificate for each Ingress Controller, and the service-ca
+controller puts the certificate in a secret named `router-metrics-certs-<name>`,
+where `<name>` is the name of the Ingress Controller, in the
+`openshift-ingress`. namespace.
+
+Each Ingress Controller has a default certificate that the Controller uses for
+secured routes that do not specify their own certificates. You can set a custom
+default certificate, or else the Operator generates a certificate for you. The
+Operator uses its own self-signed signing certificate to sign any default
+certificate that it generates. The Operator generates this signing certificate
+and puts it in a secret named router-ca in the openshift-ingress-operator
+namespace. When the Operator generates a default certificate, it puts the
+default certificate in a secret named router-certs-<name> (where <name> is the
+name of the Ingress Controller) in the openshift-ingress namespace.
+
+[WARNING]
+====
+The Operator generates a default certificate for an Ingress Controller to serve
+as a placeholder till you configure a custom default certificate. Do not use
+operator-generated default certificates in production clusters.
+====
+
+[discrete]
+== Expiration
+
+The expiration terms for the Ingress Operator's certificates are as follow:
+
+* The expiration date for metrics certificates that the `service-ca` controller
+creates is two years after the date of creation.
+* The expiration date for the Operator's signing certificate is two years after
+the date of creation.
+* The expiration date for default certificates that the Operator generates is two
+years after the date of creation.
+
+You cannot specify custom expiration terms on certificates that the Ingress
+Operator or `service-ca` controller create.
+
+You cannot specify expiration terms when installing {product-title} for
+certificates that the Ingress Operator or `service-ca` controller create.
+
+[discrete]
+== Services
+
+Prometheus uses the certificates that secure metrics.
+
+The Ingress Operator uses its signing certificate to sign default certificates
+that it generates for Ingress Controllers for which you do not set custom
+default certificates.
+
+Cluster components that use secured routes may use the default Ingress
+Controller's default certificate.
+
+Ingress to the cluster via a secured route uses the default certificate of the
+Ingress Controller via which the route is accessed unless the route specifies
+its own certificate.
+
+[discrete]
+== Updates
+
+To force regeneration of a certificate, delete the corresponding secret.
+
+[discrete]
+== Renewal
+
+The `service-ca` controller automatically rotates the certificates that it
+issues. However, it is possible to use `oc delete secret <secret>` to
+manually rotate service serving certificates.
+
+The Ingress Operator does not rotate its own signing certificate or the default
+certificates that it generates. Operator-generated default certificates are
```

Comment:
Just wondering how this last sentence is applicable in this section; it was already mentioned above.